notebooks/community/ml_ops/stage2/get_started_bqml_training.ipynb

```diff
@@ -1,1221 +1,1042 @@
 {
```

Comment:
<p>Header doesn't match latest in notebook template</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/397/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>